prisma/seed.ts

```diff
@@ -19,8 +20,25 @@ const clearConnections = async () => {
   const users = await prisma.user.findMany();
 
   await Promise.all(
-    users.map(async (user) => {
-      return await prisma.user.update({
+    users.map((user) =>
+      Promise.all([
+        prisma.invitation.findMany({
```

Comment:
Why is the ``findMany`` call here?

Reply:
multiple invitations can be sent or received by a person?

Reply:
Sure, but where is the return value getting used?